rng.info.yml

```diff
@@ -2,7 +2,7 @@ name: RNG
 type: module
 description: 'Provides the ability to create and manage events and registrations.'
 package: RNG
-core_version_requirement: ^8.8.4 || ^9.0
+core_version_requirement: ^9.4 || ^10
```

Comment:
```suggestion
core_version_requirement: ^9.4 || ^10 || ^11
```

Reply:
updated core_version_requirement for submodules too